### ISSUE TITLE
Missing AccessibilityLabel for Settings, Profile buttons.

### ISSUE THREAD

**Issue by MarcoZehe** [DagAgren/toot#122]
The buttons in the top bar, like Settings, and on the Local Timeline tab, the Profile one, don't have an AccessibilityLabel for VoiceOver. VoiceOver only says "Button".

Easiest way to fix is to call SetAccessibilityLabel on the proper image element and give it a localizable string like "Settings", "My Profile" or the like.

**Comment by DagAgren** [comment]
I think this one has been fixed.
